packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -241,21 +244,23 @@ export class MarketOperationUtils {
                 this._sampler.balancerPoolsCache,
                 this._liquidityProviderRegistry,
                 this._multiBridge,
+                this._sampler.bancorService,
             ),
             // Get buy quotes for taker -> maker.
             await DexOrderSampler.ops.getBuyQuotesAsync(
                 difference(
                     BUY_SOURCES.concat(
                         this._liquidityProviderRegistry !== NULL_ADDRESS ? [ERC20BridgeSource.LiquidityProvider] : [],
                     ),
-                    _opts.excludedSources.concat(ERC20BridgeSource.Balancer),
+                    _opts.excludedSources.concat([ERC20BridgeSource.Balancer]),
```

Comment:
`concat` works on individual elements *and* arrays because types aren't real in node 😭 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat